packages/jaeger-ui/src/constants/default-config.tsx

```diff
@@ -87,6 +87,25 @@ export default deepFreeze(
         trackErrors: true,
         customWebAnalytics: null,
       },
+      monitor: {
+        menuEnabled: true,
+        emptyState: {
+          mainTitle: 'Get started with Service Performance Monitoring',
+          subTitle:
+            'A high-level monitoring dashboard that helps you cut down the time to identify and resolve anomalies and issues.',
+          description:
+            'Service Performance Monitoring aggregates tracing data into RED metrics through a one-time configuration and visualizes these metrics through service and operation level dashboards.',
+          button: {
+            text: 'Read the Documentation',
+            onClick: () => window.open('https://www.jaegertracing.io/docs/latest/atm/'),
```

Comment:
FYI, https://github.com/jaegertracing/documentation/pull/567, so the URL will change to https://www.jaegertracing.io/docs/latest/spm/. 

Alternatively, we can make this change in another PR, as I noticed there are other references to ATM here like `MonitorATMEmptyState`.

Apologies for the back and forth!